utilities/elig-results-to-word.vbs

```diff
@@ -0,0 +1,106 @@
+'Starts by defining a changelog array
+changelog = array()
+
+'INSERT ACTUAL CHANGES HERE, WITH PARAMETERS DATE, DESCRIPTION, AND SCRIPTWRITER. **ENSURE THE MOST RECENT CHANGE GOES ON TOP!!**
+'Example: call changelog_update("01/01/2000", "The script has been updated to fix a typo on the initial dialog.", "Jane Public, Oak County")
+call changelog_update("06/02/2021", "Initial version.", "Casey Love, Hennepin County")
+
+'Actually displays the changelog. This function uses a text file located in the My Documents folder. It stores the name of the script file and a description of the most recent viewed change.
+changelog_display
+'END CHANGELOG BLOCK =======================================================================================================
+
+
+'TODO - there is currently no handling for HC
+EMConnect ""
+Call MAXIS_case_number_finder(MAXIS_case_number)
+
+Dialog1 = ""
+BeginDialog Dialog1, 0, 0, 231, 195, "Case Number to Read ELIG Results"
+  EditBox 65, 65, 50, 15, MAXIS_case_number
+  ButtonGroup ButtonPressed
+    OkButton 115, 170, 50, 15
+    CancelButton 170, 170, 50, 15
+  Text 15, 15, 175, 20, "This script works by pulling all of the information from a specific version of ELIG into word."
+  Text 15, 40, 195, 20, "Enter the Case Number and Navigate in MAXIS now to the start of the ELIG version that you would like copied."
+  Text 15, 70, 45, 10, "Case Number"
+  GroupBox 15, 90, 205, 40, "NAVIGATE IN MAXIS NOW TO THE ELIG VERSION TO COPY"
+  Text 30, 105, 105, 10, "- ELIG can be for any program."
+  Text 30, 115, 165, 10, "- Version of ELIG can be approved or unapproved."
+  Text 15, 145, 180, 20, "When the script continues, it will look for the first page of ELIG Results. If not found, the dialog will reappear."
+EndDialog
+
+Do
+	Do
+		Do
+			err_msg = ""
+
+			dialog Dialog1
+			cancel_without_confirmation
+
+			Call validate_MAXIS_case_number(err_msg, "*")
+
+		Loop until err_msg = ""
+		elig_results_program_found = ""
+		EMReadScreen MX_line_3, 78, 3, 2
+		If InStr(MX_line_3, "DWPR") Then elig_results_program_found = "DWP"
+		If InStr(MX_line_3, "MFPR") Then elig_results_program_found = "MFIP"
+		If InStr(MX_line_3, "MSPR") Then elig_results_program_found = "MSA"
+		If InStr(MX_line_3, "GAPR") Then elig_results_program_found = "GA"
+		If InStr(MX_line_3, "CAPR") Then elig_results_program_found = "Cash Denial"
+		If InStr(MX_line_3, "GRPR") Then elig_results_program_found = "GRH"
+		If InStr(MX_line_3, "FCSM") Then elig_results_program_found = "IV-E"
+		If InStr(MX_line_3, "EMPR") Then elig_results_program_found = "EMER"
+		If InStr(MX_line_3, "FSPR") Then elig_results_program_found = "SNAP"
+		If elig_results_program_found = "" Then MsgBox "MAXIS must be at the first page of Eligibility Results for this script to run." & vbCr & vbCr & "The dialog will return." & vbCr & vbCr & "NAVIGATE TO ELIG RESULTS WHILE THE DIALOG IS UP."
```

Comment:
Not sure if it's necessary but wondered about handling to ensure that the script is pulling results for the case number initially entered. I put in a case number, received the notice to move to elig results, but then navigated to elig results for a different case number. It printed the results for the new case number as expected, but thought about whether an error message of some sort might be useful if the elig results are for a different case number.

Reply:
Good thought - I think with this script we don't really care what the case number is. Removing it entirely from the dialog.